colcon_alias/argument_parser/alias.py

```diff
@@ -57,6 +68,10 @@ def _add_aliases(self):
         if not config:
             return
 
+        # Aliases may not override verbs
+        for existing_verb in self._subparser._verbs:
+            config.pop(existing_verb, None)
```

Comment:
I think it is worth reporting a warning when this happens (which should be never) so that users are not surprised that they got a vanilla command rather than an alias.

Reply:
Sure, added a warning in b8bd323.